app/gui/view/component-browser/component-list-panel/grid/src/lib.rs

```diff
@@ -145,6 +145,7 @@ ensogl_core::define_endpoints_2! {
         model_for_entry(GroupEntryId, EntryModel),
         switch_section(SectionId),
         switch_section_no_animation(SectionId),
+        set_top_section_count(usize),
```

Comment:
I think this should go in content::Info, as we shall update the top section count every time the content is refreshed.